css/style.css

```diff
@@ -1,221 +1,206 @@
-body,
-div,
-ul,
-li,
-p,
-h1,
-h2,
-h3,
-h4,
-h5,
-h6 {
-  padding: 0;
-  margin: 0;
-  font-family: "Roboto", sans-serif;
+body, div, ul, li, p, h1, h2, h3, h4, h5, h6 {
```

Comment:
Good use of organizing selectors that share similar property declarations. Nice.